src/Jackett.Common/Indexers/SubsPlease.cs

```diff
@@ -0,0 +1,149 @@
+            => query.IsTest || string.IsNullOrWhiteSpace(query.SearchTerm)
+            ? await FetchNewReleases()
+            : await PerformSearch(query);
+
+        private async Task<IEnumerable<ReleaseInfo>> PerformSearch(TorznabQuery query)
+        {
+            var queryParameters = new NameValueCollection
+            {
+                { "f", "search" },
+                { "tz", "America/New_York" },
+                { "s", query.SearchTerm }
+            };
+            var response = await RequestWithCookiesAndRetryAsync(SiteLink + "/api/?" + queryParameters.GetQueryString());
+            if (response.Status != HttpStatusCode.OK)
+                throw new WebException($"SubsPlease search returned unexpected result. Expected 200 OK but got {response.Status}.", WebExceptionStatus.ProtocolError);
+
+            var results = ParseApiResults(response.ContentString);
+            return results.Where(release => query.MatchQueryStringAND(release.Title));
+        }
+
+        private async Task<IEnumerable<ReleaseInfo>> FetchNewReleases()
+        {
+            var queryParameters = new NameValueCollection
+            {
+                { "f", "latest" },
+                { "tz", "America/New_York" }
+            };
+            var response = await RequestWithCookiesAndRetryAsync(SiteLink + "/api/?" + queryParameters.GetQueryString());
+            if (response.Status != HttpStatusCode.OK)
+                throw new WebException($"SubsPlease search returned unexpected result. Expected 200 OK but got {response.Status}.", WebExceptionStatus.ProtocolError);
+
+            return ParseApiResults(response.ContentString);
+        }
+
+        private List<ReleaseInfo> ParseApiResults(string json)
+        {
+            var releaseInfo = new List<ReleaseInfo>();
+            var releases = JsonConvert.DeserializeObject<Dictionary<string, Release>>(json);
+            foreach (var keyValue in releases)
+            {
+                Release r = keyValue.Value;
+                var baseRelease = new ReleaseInfo
+                {
+                    Details = new Uri(SiteLink + $"shows/{r.Page}/"),
+                    PublishDate = r.Release_Date.DateTime,
+                    Files = 1,
+                    Category = new List<int> { TorznabCatType.TVAnime.ID },
+                    Size = 0,
+                    Seeders = 1,
+                    Peers = 2,
+                    MinimumRatio = 1,
+                    MinimumSeedTime = 172800, // 48 hours
+                    DownloadVolumeFactor = 0,
+                    UploadVolumeFactor = 1
+                };
+                foreach (var d in r.Downloads)
+                {
+                    var release = (ReleaseInfo)baseRelease.Clone();
+                    //Ex: [SubsPlease] Shingeki no Kyojin (The Final Season) - 64 (1080p)
+                    release.Title += $"[SubsPlease] {r.Show} - {r.Episode} ({d.Res}p)";
```

Comment:
I don't know much about anime releases. It's common to add the site in each release? Does it work with Sonarr?

Reply:
Having the group that created the release be the first component of the name is standard for anime (see nyaa.si). Many groups may make releases for any given anime, so it's important to be able to search by and distinguish among them. It does work with Sonarr, yes. Similar logic was part of the now defunct HorribleSubs indexer.

Reply:
Hmm, actually, I think there is something missing here, thank you. Search queries that include [SubsPlease] that are sent to this indexer should match, they're not currently because they're not part of the title returned on the site. It's likely enough to check if that string is at the beginning of the query and remove it. (Going to check how HorribleSubs handled this)

Reply:
Fixed.